build/dependencies.props

```diff
@@ -13,5 +13,6 @@
     <SerilogExtensionsLoggingVersion>1.4.0</SerilogExtensionsLoggingVersion>
     <SerilogFileSinkVersion>3.2.0</SerilogFileSinkVersion>
     <SystemReflectionMetadataVersion>1.5.0-*</SystemReflectionMetadataVersion>
+    <SystemServiceProcessVersion>4.5.0-*</SystemServiceProcessVersion>
```

Comment:
Should we specify the package name completely? E.g. `SystemServiceProcessServiceControllerVersion`. If this setting is desired/necessary at all.

Reply:
> Should we specify the package name completely?

Either way is fine. I'll be deleting this file soon anyways as we move to a system called "lineups" to manage our package versions.

Any reason you need 4.5.0-* specifically? Or will 4.4.0 work?

cref https://github.com/aspnet/Universe/blob/dev/build/dependencies.props#L114

Reply:
Yeah, I saw a PR of you coming by where you implemented this. Figured it would be fine.

I've used the 4.5.0 packages because the API was added in .NET Core 2.1: https://github.com/dotnet/corefx/pull/22920. I thought this maps to 4.5.0-* packages of the runtime.

Reply:
Seconded, it's easier if this can be 4.4.0, then we'll update to 4.5.0 at a later date in all repos at once.

Reply:
The 4.4.0 package only has a few types from System.ServiceProcess.* namespace. 4.5.0 adds more. Which version we need will depends which API is actually referenced from hosting. I haven't looked closely at what we use. If we need 4.5.0 that's no problem, we just need to setup our internal build processes to start using new packages from the corefx team.

Reply:
This package [uses `ServiceBase`](https://github.com/aspnet/Hosting/blob/9b1cbedffcc48154c7c52655efe67927ae933e32/src/Microsoft.AspNetCore.Hosting.WindowsServices/WebHostService.cs#L12), which was added in .NET Core 2.1.

Reply:
Good to know. Then yes, we'll need the 4.5.0 version of the package.